hwconf/hw_Cheap_FOCer_2.h

```diff
@@ -0,0 +1,306 @@
+/*
+	Copyright 2016 Benjamin Vedder	benjamin@vedder.se
+
+	This file is part of the VESC firmware.
+
+	The VESC firmware is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    The VESC firmware is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+    */
+
+#ifndef HW_Cheap_FOCer_2
+#define HW_Cheap_FOCer_2
+
+#define HW_NAME					"Cheap FOCer 2"
```

Comment:
Is this only version 0.9 or also 1.0?

Reply:
Its only 1.0 beta testers can either build their own firmware, or swap 2 resistors.

Reply:
Do you have information on which two resistors to swap?

Reply:
R64 & R65 need to be reduced from 20k to 10k, It wont give you a full v1, but it will make v1 firmware compatible, and fix the DRV errors that require a reboot. I might make a guide with images.